recipe/meta.yaml

```diff
@@ -39,6 +41,7 @@ about:
   home: https://jcrist.github.io/skein/
   license: BSD-3-Clause
   license_family: BSD
+  license_file: {{ SP_DIR }}/{{ name }}-{{ version }}.dist-info/LICENSE
```

Comment:
This is a little hacky. However as installing the `whl` does place the license file in `site-packages`, this is a reliable way to include it in the Conda package without extracting the `whl` manually and rummaging around for this file.